examples/proxy-server/src/main/java/example/armeria/proxy/Main.java

```diff
@@ -0,0 +1,72 @@
+package example.armeria.proxy;
+
+import java.time.Duration;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import com.linecorp.armeria.server.Server;
+import com.linecorp.armeria.server.ServerBuilder;
+import com.linecorp.armeria.server.ServerCacheControl;
+import com.linecorp.armeria.server.file.HttpFileBuilder;
+import com.linecorp.armeria.server.healthcheck.HttpHealthCheckService;
+import com.linecorp.armeria.server.logging.LoggingService;
+
+public final class Main {
+
+    private static final Logger logger = LoggerFactory.getLogger(Main.class);
+
+    public static void main(String[] args) throws Exception {
+        final Server backend1 = newBackendServer(8081, 500);
+        final Server backend2 = newBackendServer(8082, 250);
+        final Server backend3 = newBackendServer(8083, 100);
+        backend1.start().join();
+        backend2.start().join();
+        backend3.start().join();
+
+        final Server proxyServer = newProxyServer(8080, 8443);
+
+        Runtime.getRuntime().addShutdownHook(new Thread(() -> {
+            backend1.stop().join();
+            backend2.stop().join();
+            backend3.stop().join();
+
+            proxyServer.stop().join();
+            logger.info("The proxy server has been stopped.");
+        }));
+
+        proxyServer.start().join();
+        logger.info("The proxy server has been started.");
```

Comment:
nit: How about printing the webpage link like what I did [here](https://github.com/line/armeria/blob/master/examples/grpc-service/src/main/java/example/armeria/grpc/Main.java#L35)?